docs/bsp.md

```diff
@@ -352,10 +352,13 @@ trait ShowMessageParams {
   def `type`: Int
 
   /** The message id. */
-  def id: Long
+  def id: string
```

Comment:
Should this be called `taskId` instead? And `parentTaskId`, respectively. The docstrings should have the explanation below on what the difference between "task id" and "request id" is.

Reply:
It would also be good to mention that this is is generated by the server, not the client.

Reply:
Should this be optional?